Lombiq.VueJs/Extensions/QueryExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using OrchardCore.Settings;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace YesSql;
+
+public static class ControllerExtensions
+{
+    public static async Task<object> GetPageFromQueryAsync<T>(

```

Comment:
I mean that tuple won't work because of the JSON. It serializes differently:
![image](https://user-images.githubusercontent.com/4007293/160853821-7775efea-9d38-4de9-9c23-42f0373a4d54.png)

Reply:
That's unfortunate. Anyway, the problem still stands, so let's return a POCO, then.